samples/wifi/softap/Kconfig

```diff
@@ -0,0 +1,58 @@
+source "Kconfig.zephyr"
+
+menu "Nordic SoftAP sample"
+
+config SOFTAP_SAMPLE_REG_DOMAIN
+	string "The ISO/IEC alpha2 country code"
+	default "IN"
+	help
+	  Set the ISO/IEC alpha2 country code for the country in
+	  which the device is currently operating.
+	  For world regulatory, use "00".
+
+config SOFTAP_SAMPLE_FREQUENCY_BAND
+	int "SoftAP frequency band"
+	default 0 if SOFTAP_SAMPLE_2_4GHz
+	default 1 if SOFTAP_SAMPLE_5GHz
+	help
+	  Set the Wi-Fi frequency band for SoftAP operation.
+
+config SOFTAP_SAMPLE_2_4GHz
+	bool "2.4GHz band"
+	default y
+	help
+	  Enable this option if the SoftAP should operate in the 2.4GHz band.
+
+config SOFTAP_SAMPLE_5GHz
+	bool "5GHz band"
+	default n
```

Comment:
This line can be removed, default is no. and may be add a dependency on reg domain here? or at least expand the note/help.